information-schema/information-schema-tikv-region-status.md

```diff
@@ -39,7 +39,27 @@ DESC tikv_region_status;
 17 rows in set (0.00 sec)
 ```
 
-You can implement the `top confver`, `top read` and `top write` operations in pd-ctl via the `ORDER BY X LIMIT Y` operation on the `EPOCH_CONF_VER`, `WRITTEN_BYTES` and `READ_BYTES` columns.
+The descriptions of the columns in the `TIKV_REGION_STATUS` table are as follows:
+
+* `REGION_ID`: The ID of the Region.
+* `START_KEY`: The value of the start key of the Region.
+* `END_KEY`: The value of the end key of the Region.
+* `TABLE_ID`: The ID of the table to which the Region belongs.
+* `DB_NAME`: The name of the database to which `TABLE_ID` belongs.
+* `TABLE_NAME`: The name of the table to which the Region belongs.
+* `IS_INDEX`: Whether the Region data is an index. 0 means that it's not an index, while 1 means that it's an index. If the current Region contains both table data and index data, there will be multiple rows of records, and `IS_INDEX` is 0 and 1 respectively.
+* `INDEX_ID`: The ID of the index to which the Region belongs. If `IS_INDEX` is 0, the value of this column is NULL.
+* `INDEX_NAME`: The name of the index to which the Region belongs. If `IS_INDEX` is 0, the value of this column is NULL.
+* `EPOCH_CONF_VER`: The version number of the Region configuration. The version number increases when a peer is added or removed.
+* `EPOCH_VERSION`: The current version number of the Region. The version number increases when the Region is split or merged.
+* `WRITTEN_BYTES`: The amount of data written to the Region (bytes).
+* `READ_BYTES`: The amount of data (bytes) that has been read from the Region.
+* `APPROXIMATE_SIZE`: The approximate data size (MB) of the Region.
+* `APPROXIMATE_KEYS`: The approximate number of keys in the Region.
+* `REPLICATIONSTATUS_STATE`: The current replication status of the Region. The status might be `UNKNOWN`, `SIMPLE_MAJORITY`, or `INTEGRITY_OVER_LABEL`.
+* `REPLICATIONSTATUS_STATEID`: The identifier corresponding to `REPLICATIONSTATUS_STATE`.
+
+Also, you can implement the `top confver`, `top read` and `top write` operations in pd-ctl via the `ORDER BY X LIMIT Y` operation on the `EPOCH_CONF_VER`, `WRITTEN_BYTES` and `READ_BYTES` columns.
```

Comment:
What's the source of this information? It is not in the [latest Chinese doc](https://docs.pingcap.com/zh/tidb/dev/system-table-information-schema#tikv_region_status-%E8%A1%A8).

Reply:
The English version is the latest version (zh doc is left behind because of https://github.com/pingcap/docs/pull/3178/files#diff-26d8635bed25d9d743978efd897c83baR42)